docs/anomaly-detection/QuickStart.md

```diff
@@ -0,0 +1,102 @@
+---
+sort: 1
+weight: 1
+title: VictoriaMetrics Anomaly Detection Quick Start
+menu:
+  docs:
+    parent: "anomaly-detection"
+    weight: 1
+    title: Quick Start
+aliases:
+- /anomaly-detection/QuickStart.html
+---
+
+# `vmanomaly` Quick Start
```

Comment:
should be the same as title `title: VictoriaMetrics Anomaly Detection Quick Start`